package.json

```
@@ -1,42 +1,48 @@
 {
   "name": "lz-string",
+  "author": {
+    "name": "pieroxy",
+    "email": "pieroxy@pieroxy.net"
+  },
+  "contributors": [
+    {
+      "name": "Karnthis",
+      "email": "me@erinrivas.com"
+    }
+  ],
   "version": "1.5.0",
```

Suggesting this as an alpha tag so people don't automatically update to it - once code formatting is sorted and we're happy with "this hasn't changed or broken anything including imports" then pull it to full release

```suggestion
  "version": "1.6.0-alpha.1",
```

---

sure I'll flag it as an RC then.